hub.js

```diff
@@ -53,7 +53,7 @@ const randomize = (lower, upper) => {
 
 // Create random port as starting point
 
-let pid = randomize(1000, 65535);
+//let pid = randomize(1000, 65535);
```

Comment:
This globally-scoped variable set itself once and then never was overwritten; all of the other references seemed to ignore the global scope or create local versions of the `pid` variable.